pages/nft/iscn/_iscnId.vue

```diff
@@ -802,11 +800,20 @@ export default class NFTTestMintPage extends Vue {
       const sendMessages = nfts.map((i) =>
         formatMsgSend(this.address, LIKER_NFT_API_WALLET, this.classId, i.id),
       )
-      let messages = mintMessages;
-      if (this.isWritingNFT) messages = messages.concat(sendMessages);
-
       logTrackerEvent(this, 'IscnMintNFT', 'MintNFT', this.classId, 1);
-      this.mintNFTResult = await signingClient.sendMessages(this.address, messages)
+      if (this.isUsingLikerLandApp) {
+        let res;
+        res = await signingClient.sendMessages(this.address, mintMessages)
+        if (this.isWritingNFT) res = await signingClient.sendMessages(this.address, sendMessages)
```

Comment:
So Liker Land app will sign twice, one for minting and transfer in WNFT flow?

Reply:
yes as to reduce the size
potentially it will cause recovery issue where user kept on minting nft but didn't send the nft away, we can add an extra state to handle that though